flume-ng-sinks/flume-hdfs-sink/src/test/java/org/apache/flume/sink/hdfs/TestHDFSEventSink.java

```diff
@@ -1323,6 +1323,64 @@ public void testCloseOnIdle() throws IOException, EventDeliveryException, Interr
     fs.close();
   }
 
+  @Test
```

Comment:
Please add a comment at the top of the test with a description. You can leave out @throws clauses and all of that junk that is not really additionally helpful to developers reading the test code